File: Practice Sorting/5_Facebook.py
t = int(input())
for _ in range(t):
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))
    # your code goes here
    max_comments=0
    max_val=0
    position=0
    for index,value in enumerate(a):
        if value>max_val:
            max_val=value
            max_comments=b[index]
            position=index+1
        elif value==max_val:
            if b[index]>b[position-1]:
                max_comments=b[index]
                position=index+1
    print(position)

File: Practice Sorting/codigo_cs/5Facebook.cs
using System;

public class Test5
{
    public static void Facebook()
    {
        // your code goes here
        int t = int.Parse(Console.ReadLine());
        for (int i = 0; i < t; i++)
        {
            int n = int.Parse(Console.ReadLine());
            int[] a = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
            int[] b = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
            // your code goes here
            int maxComments = 0;
            int maxVal = 0;
            int position = 0;
            for (int index = 0; index < n; index++)
            {
                if (a[index] > maxVal)
                {
                    maxVal = a[index];
                    maxComments = b[index];
                    position = index + 1;
                }
                else if (a[index] == maxVal)
                {
                    if (b[index] > b[position - 1])
                    {
                        maxComments = b[index];
                        position = index + 1;
                    }
                }
            }
            Console.WriteLine(position);
        }
    }
}